js/web/i18n/ro.js

```diff
@@ -0,0 +1,398 @@
+/*
+ * **************************************************************************************
+ *
+ * Dateiname:                 en.js
+ * Projekt:                   foe-chrome
+ *
+ * erstellt von:              Daniel Siekiera <daniel.siekiera@gmail.com>
+ * erstellt am:	              17.12.19, 22:44 Uhr
+ * zuletzt bearbeitet:       17.12.19, 22:18 Uhr
+ *
+ * Copyright © 2019
+ *
+ * **************************************************************************************
+ */
+
+let i18n = {
+	"Local" : "ro-RO",
+	"DateTime" : "D/M/YY h:mm:ss a",
+
+	"Boxes" : {
+		"OwnpartCalculator" : {
+			"Title": "Calculator de contribuție proprie",
+			"HelpLink": "https://foe-rechner.de/extension/index#Eigenanteilsrechner",
+            "Step": "Nivel",
+            "OldLevel": "Nivelul anterior",
+			"PatronPart": "Contribuția donatorilor",
+			"OwnPart": "Contribuția proprie",
+            "LGTotalFP": "Total PF nivel",
+            "OwnPartRemaining": "Rest de depus",
+			"Done": "Depus",
+			"BPs": "BPs",
+			"Meds": "Meds",
+			"Ext": "Ext",
+			"Arc": "Arc",
+			"Order": "Ordine",
+			"Deposit": "Depozit",
+			"CopyValues": "Copiază valorile",
+			"Note": "Notă",
+			"YourName": "Numele tău",
+			"IndividualName": "Denumire CL",
+			"OutputScheme": "Șablon",
+            "Auto": "Auto",
+            "Place": "Locul",
+            "Levels": "Niveluri",
+            "NoPlaceSafe": "Nici un loc nu este asigurat"
+		},
+
+		"Calculator": {
+			"Title": "Calculator costuri",
+			"HelpLink": "https://foe-rechner.de/extension/index#Kostenrechner",
+			"Step": "Nivelul ",
+			"AvailableFP": "Puncte forge disponibile",
+			"ArcBonus": "Bonusul Domului",
+			"Earnings": "Recompensă",
+			"Rate": "Procent",
+			"Up2LevelUp": "PF rămase până la inchidere",
+			"FP": "PF",
+			"Save": "Salvat",
+			"Commitment": "Cost",
+			"Profit": "Profit",
+			"LevelWarning": "ATENȚIE! Închide nivelul la CL!",
+			"NoFPorMedsAvailable": "PF sau planuri indisponibile",
+			"LGNotOpen": "Următorul nivel nu este deblocat",
+			"LGNotConnected": "Clădirea nu este conectată la stradă",
+			"ActiveRecurringQuest": Misiunea repetitivă actuală:",
```

Comment:
Before 'Misiunea' a double quote is missing. The Rest of your translations looking fine.

Reply:
The problem was solved.